packages/core/forms/src/components/forms/RLAForm.vue

```diff
@@ -337,7 +337,9 @@ const scopingSchema = computed(() => {
     return undefined
   }
 
-  return { fields: [selectionGroup] }
+  const enableSwitch = props.formSchema?.fields.find((field: any) => field.pinned && field.type === 'switch' && field.model === 'enabled')
```

Comment:
nit: Could this be simplified to something like:

```suggestion
  const enableSwitch = props.formSchema?.fields.find((field: any) => field.model === 'enabled')
```